spec/README.md

```diff
@@ -67,6 +67,7 @@ Example structure for the German language file, which only contains lower case c
         "alphabet": "Latn",
         "continent": "EU",
         "language": "German",
+        "variant": "",
```

Comment:
I think this wouldn't be compatible with [this change](https://github.com/diacritics/database/pull/7/commits/925cc762a80829c9ac7f021f0f613dce48c4291b) as `variant` is optional, but if it's defined it requires at least one character. If there's no variant I'd vote to not set this property.